dev/com.ibm.ws.jca.utils/src/com/ibm/ws/jca/utils/metagen/MetatypeGenerator.java

```diff
@@ -415,6 +416,10 @@ private void buildResourceAdapters(MetaGenInstance instance) throws ResourceAdap
             }
         }
 
+        if (!adapter.getSecurityPermissions().isEmpty()) {
```

Comment:
Did something change to require this check where it wasn't required before, or is this a bug fix?

Reply:
That looks like a bad merge.  Reviewing the file history ...

Reply:
That was removed by:
https://github.com/tbitonti/open-liberty/commit/8a1a5ac95182785183d4d022823ea7d35d4742d4#diff-9afe53d563716b58e4fba3b516e9f09e
I'm taking it back out.